open-craft/labxchange-xblocks
# [FAL-2654] [LX-2395] Add HtmlBlock

This pull request is about preparing the new HtmlBlock.

**Jira tickets**

- [FAL-2654](https://tasks.opencraft.com/browse/FAL-2654)
- [LX-2395](https://tasks.opencraft.com/browse/LX-2395)

**Testing instructions**

- You need to checkout your `edx-platform` devstack to this branch: https://github.com/edx/edx-platform/pull/29517
- You need to use this `labxchange-dev` branch: https://gitlab.com/opencraft/client/LabXchange/labxchange-dev/-/merge_requests/1578/diffs?commit_id=72d899ba2aca22200123f27f4f2837cafec8b741#1ab62691cdbe5e874dea65d10749e0477817a971 
- You need to create a `MIGRATION_TESTERS` user group in LabXchange and add your testing user to it
- Run `pip install -e /edx/src/labxchange-xblocks` in your lms container to update it
- Revert to change you may have done to `/edx/app/edxapp/edx-platform/common/lib/xmodule/setup.py`
- Restart your LMS container
- Try to open existing text assets
- Try to create new text assets
- Check the LMS logs to be sure that LabXchange is doing lms request with the `?lx_block_types=1` parameter

**Reviewers**
